components/components.yaml

```diff
@@ -938,4 +938,15 @@
   Category: Other
   ProjectUrl: https://github.com/Particular/NServiceBus.UniformSession
   NugetOrder:
-    - NServiceBus.UniformSession
\ No newline at end of file
+    - NServiceBus.UniformSession
+
+  Key: GatewayChannelHttpVNext
+  Name: HttpVNext Gateway Channel
+  DocoUrl: nservicebus/gateway/httpvnext-channel
+  GitHubOwner: welshdave
+  Category: Other
+  SupportLevel: Community
+  LicenseUrl: https://github.com/welshdave/NServiceBus.Gateway.Channels.HttpVNext/blob/master/LICENSE
+  ProjectUrl: https://github.com/welshdave/NServiceBus.Gateway.Channels.HttpVNext
+  NugetOrder:
+    - NServiceBus.Gateway.Channels.HttpVNext
```

Comment:
@Particular/docs-maintainers the build server blow up with 

`System.Exception : Could not find nuget for id:NServiceBus.Gateway.Channels.HttpVNext`

but the nuget name seems correct to me? (https://www.nuget.org/packages/NServiceBus.Gateway.Channels.HttpVNext)